.azure/build-linux-images.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+  images: []
+
+jobs:
+- ${{ each img in parameters.images }}:
+  - job: build_${{ img.name }}_${{ img.version }}
+    displayName: Build ci-${{ img.name }}_${{ img.version }}
+    dependsOn: ${{ img.dependsOn }}
+    timeoutInMinutes: 0
+    condition: ${{ img.condition }}
+    variables:
+      DOCKER_CLI_EXPERIMENTAL: enabled
+      DOCKER_BUILDKIT: '1'
+    pool:
+      vmImage: 'ubuntu-16.04'
```

Comment:
It probably doesn't make a big difference, but Azure Pipelines now officially supports Ubuntu 18.04.3 hosted agents, the version of Docker CE is the same (3.0.8), but could be worth moving to the newer Ubuntu LTS builder. https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md